lib/vscode/src/vs/server/node/cliProcessMain.ts

```diff
@@ -0,0 +1 @@
+../../code/node/cliProcessMain.ts
```

Comment:
Should we add this one to the rule instead? I think it should be safe to call `code/node` code from `server/node`.